kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx

```diff
@@ -82,7 +82,7 @@ function renderVersionAction(version, nothingToCommitDiff, app, history, deployV
         <button
           className={classNames("btn", { "secondary dark": isRollback, "secondary blue": isSecondaryBtn, "primary blue": isPrimaryButton })}
           disabled={version.status === "deploying"}
-          onClick={() => needsConfiguration ? history.push(`/app/${app.slug}/config/${version.sequence}`) : deployVersion(version)}
+          onClick={() => needsConfiguration ? history.push(`/app/${app.slug}/config/${version.sequence}`) : isRollback ? deployVersion(version, true) : deployVersion(version)}
```

Comment:
```suggestion
          onClick={() => needsConfiguration ? history.push(`/app/${app.slug}/config/${version.sequence}`) : deployVersion(version, isRollback)}
```

Reply:
I'm not sure it's this simple. the `deployVersion` function is passed to it and can be different. I only want to pass that var if it is rollback

Reply:
ah i see.